src/core/server.h

```diff
@@ -129,6 +130,18 @@ class InferenceServer {
   float TensorFlowGPUMemoryFraction() const { return tf_gpu_memory_fraction_; }
   void SetTensorFlowGPUMemoryFraction(float f) { tf_gpu_memory_fraction_ = f; }
 
+  // Get / set Tensorflow vGPU memory limits
+  std::map<int, std::vector<float>> TensorFlowVGPUMemoryLimits() const
```

Comment:
Strange to have a vector, why not a std::pair<int, uint64_t>?

Reply:
I have to turn it into a vector somewhere. I suppose I could do it when constructing the backend config?